query/src/users/auth/auth_mgr.rs

```diff
@@ -33,6 +33,7 @@ pub struct AuthMgr {
 pub enum Credential {
     Jwt {
         token: String,
+        hostname: Option<String>,
```

Comment:
Why JWT also needs hostname?

Reply:
If use jwt to do auth, it seems we need `client_ip` to check it's from localhost or not, so I add hostname field.